.github/workflows/stale_issues_and_pr_cleanup.yml

```diff
@@ -0,0 +1,30 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+permissions:
+  issues: write
+  pull-requests: write
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    env:
+      DAYS_BEFORE_ISSUE_STALE: 180
+      DAYS_BEFORE_PR_STALE: 180
+      DAYS_BEFORE_CLOSE: 14
+    steps:
+      - uses: actions/stale@v7
+        with:
+          days-before-issue-stale: ${{ env.DAYS_BEFORE_ISSUE_STALE }}
+          days-before-pr-stale: ${{ env.DAYS_BEFORE_PR_STALE }}
+          days-before-close: ${{ env.DAYS_BEFORE_CLOSE }}
+          stale-issue-message: 'This issue has not had any activity in ${{ env.DAYS_BEFORE_ISSUE_STALE }} days.  Cypress evolves quickly and the reported behavior should be tested on the latest version of Cypress to verify the reported behavior is still occurring.  It will be closed in ${{ env.DAYS_BEFORE_CLOSE }} days if no updates are provided'
+          stale-pr-message: 'This PR has not had any activity in ${{ env.DAYS_BEFORE_PR_STALE }} days.  If no activity is detected in the next ${{ env.DAYS_BEFORE_CLOSE }} days this ticket will be closed.'
+          stale-issue-label: 'stale'
+          stale-pr-label: 'stale'
+          close-issue-message: 'This issue has been closed due to inactivity.'
+          close-pr-message: 'This PR has been closed due to inactivity'
+          exempt-issue-labels: 'type: feature,type: enhancement,routed-to-e2e,routed-to-ct,routed-to-tools,routed-to-cloud,Popular Issue'
+          exempt-pr-labels: 'type: feature,type: enhancement,Popular Issue'
+          exempt-all-milestones: true
+          debug-only: true
```

Comment:
Seems like we should also increase the [operations-per-run](https://github.com/actions/stale#operations-per-run) to ensure we are capturing all the issues/PRs while in [debug](https://github.com/actions/stale#debugging) mode.

Reply:
added in latest change